legend-engine-application-query/src/main/java/org/finos/legend/engine/application/query/model/Query.java

```diff
@@ -32,6 +32,7 @@
     public String content;
     // We make it clear that we only allow a single owner
     public String owner;
+    public Long lastUpdateTimeStamp;
```

Comment:
Let's do `lastUpdateAt`, also let's add `createdAt` timestamp

Reply:
Similar to SDLC, let's expose these fields to the search API
<img width="1213" alt="image" src="https://user-images.githubusercontent.com/13574879/226064104-2d1223a2-9f60-4fa2-9b0a-4226f8d9d688.png">

Another thing you need to add is what can you modify the search API so that you can fetch `lastest 10 queries`, `latest 20 queries`, base on `lastUpdatedAt` - maybe introduce a boolean in the payload?